core/src/main/java/io/confluent/rest/ApplicationServer.java

```diff
@@ -166,6 +168,25 @@ private void attachMetricsListener(Metrics metrics, Map<String, String> tags) {
     }
   }
 
+  private void addJettyThreadPoolMetrics(Metrics metrics, Map<String, String> tags) {
+    //add metric for jetty thread pool queue size
+    String requestQueueSizeName = "thread-pool-queue-size";
```

Comment:
In `DbRequestMetrics.java`, we have `request-queue-size`, `active-task-count`, and `thread-pool-usage`. Why are these names different? Why don't we have a usage metric? (`getBusyThreads() / getThreads()`?)

Reply:
Updated `thread-pool-queue-size` => `request-queue-size` and add `thread-pool-usage` metric..
I have not changed `busy-thread-count` metric name as it different from `active-task-count` in jetty thread pool implementation.